src/Elcodi/Plugin/ProductCsv/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "elcodi-plugins/product-csv",
+    "license": "MIT",
+    "type": "elcodi-plugin",
+    "description": "Elcodi plugin to add Product import/export from .csv files",
+    "authors": [
+        {
+            "name": "Berny Cantos",
+            "email": "be@rny.cc"
+        },
+        {
+            "name": "Elcodi Community",
+            "homepage": "https://github.com/elcodi-plugin/product-csv/contributors"
+        }
+    ],
+    "require": {
+        "goodby/csv": "~1.0",
```

Comment:
Those are direct dependencies for `elcodi/plugin-bundle`, not for this plugin. Should we require dependencies of dependencies?